### ISSUE TITLE
update dashing.json for new Object API format

### ISSUE THREAD

**Issue by jasisk** [nextrevision/kubernetes-docset-generator#1]
Hi there! Thanks for this project (and the associated ones you made!).

Figured I'd file an issue in case you've already done the work locally—any chance you could update the dashing.json selectors for the new API format introduced with 1.5 (and carried through to 1.6 and 1.7)?

Thanks again!

**Comment by nextrevision** [comment]
Yeah, sure thing! Probably a good time to do so with KubeCon going on too ;). I'll see about this tomorrow.

**Comment by nextrevision** [comment]
They changed up quite a bit with those releases and it's going to take a bit longer than I expected. I'm going to have update a decent bit of the mappings. That's the downside of scraping. Will try and hammer at it this weekend.

**Comment by jasisk** [comment]
Oh, definitely—it's significantly different. Totally get that it's a pain. 😀
I appreciate you taking a look at it. If I have some spare cycles, I might take a crack, too.

**Comment by nextrevision** [comment]
If you get to it before me this weekend, please open a PR with what you end up with!
